app/default/application_routes.py

```diff
@@ -134,13 +138,15 @@ def tasklist(application_id):
     fund_data = get_fund_data(
         fund_id=application.fund_id,
         language=application.language,
-        as_dict=True,
+        as_dict=False,
```

Comment:
Why is this now false?

Reply:
Because we want it to return `Fund` object. Function implemented incorrectly previously https://github.com/communitiesuk/funding-service-design-frontend/pull/366/files#diff-25ffeea551d125947892e07d498380fe030e5f7c005ba15e165622de5d567f0eR184